sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/MicrosoftTeamsUserIdentifier.java

```diff
@@ -131,9 +136,9 @@ public boolean equals(Object that) {
             return false;
         }
 
-        return id == null
-            || thatId.id == null
-            || thatId.id.equals(this.id);
+        return getRawId() == null
```

Comment:
no biggy, but is not it better to just use `rawId` instead of `getRawId()`?

Reply:
It is Java. Short names are not acceptable. :-)

By Java convention, get/set Noun is the convention. I don't want to get into religious arguments.